src/pages/requests/index.tsx

```diff
@@ -129,16 +129,6 @@ export function Requests() {
     .map(groupedByCategoriesToTableData)
     .sort((a, b) => Number(b.value) - Number(a.value));
 
-  // TODO: move this logic to map component - it should get filters via context and process them accordingly
-  // TODO: fix filter mapping - it should use category_id
-  // TODO: add some comments / typing explaining what kind of black the magic is happening here :)
-  const layerFilterCategory = activeFilterItems.length
-    ? ["in", ["get", "category"], ["array", ["literal", activeFilterItems]]]
-    : ["==", ["get", "category"], "ALL"];
-
-  const layerFilterDate = activeDateFilter ? ["==", ["get", "date"], ["string", activeDateFilter]] : ["boolean", true];
-  const layerFilter = ["all", layerFilterCategory, layerFilterDate];
```

Comment:
i'm confused, this totally breaks current filter logic, why did you send this change?

Reply:
This actually fixes it. This is what I tried to explain when we're talking. We don't need that, we need proper DATA. :)
No more, no less.

Reply:
We filter data BEFORE it enters the map module. We don't need to filter it AGAIN.

Reply:
But I understand your confusion, I was confused too!

Reply:
ah, wtf, geojson is filtered data.. ok, let me see this working with my own eyes before merging